app/views/shopping_shared/tabs/_producers.html.haml

```diff
@@ -6,6 +6,9 @@
           = t :shopping_producers_of_hub, hub: '{{ shopfront.name }}'
         %ul.small-block-grid-1.medium-block-grid-2.large-block-grid-3
           %li{"ng-repeat" => "enterprise in shopfront.producers"}
-            %enterprise-modal
+            %enterprise-modal{"ng-if": "enterprise.visible != 'hidden'"}
```

Comment:
I just checked a production database:

```rb
Enterprise.distinct.pluck(:visible)
=> ["t", "public", "only_through_links", "hidden", "f"]
```

I think that `t == public` and `f == only_through_links`. So the check is right here. But then I wondered why we hide the modal?